specs-go/config.go

```diff
@@ -372,12 +390,16 @@ type LinuxResources struct {
 	// Limits are a set of key value pairs that define RDMA resource limits,
 	// where the key is device name and value is resource limits.
 	Rdma map[string]LinuxRdma `json:"rdma,omitempty"`
+	// VTPM configuration
+	VTPMs []LinuxVTPM `json:"vtpms,omitempty"`
 }
 
 // LinuxDevice represents the mknod information for a Linux special device file
 type LinuxDevice struct {
 	// Path to the device.
 	Path string `json:"path"`
+	// Path of passed-through device on host
+	Devpath string `json:"devpath"`
```

Comment:
What is this needed for?

Reply:
So that /dev/tpm10 on the host can appear as /dev/tpm0 inside the container.

Reply:
Do we need to know the host path or the entity creating the config.json can get the host device major / minor? For e.g. docker/podman have this feature:
```
sudo podman run --rm -it --device=/dev/null:/dev/mynull fedora:32 sh
sh-5.0# ls -l /dev/
total 0
crw--w----. 1 root tty  136, 0 Aug  6 21:35 console
lrwxrwxrwx. 1 root root     11 Aug  6 21:35 core -> /proc/kcore
lrwxrwxrwx. 1 root root     13 Aug  6 21:35 fd -> /proc/self/fd
crw-rw-rw-. 1 root root   1, 7 Aug  6 21:35 full
drwxrwxrwt. 2 root root     40 Aug  6 21:35 mqueue
crw-rw-rw-. 1 root root   1, 3 Aug  6 21:35 mynull
crw-rw-rw-. 1 root root   1, 3 Aug  6 21:35 null
lrwxrwxrwx. 1 root root      8 Aug  6 21:35 ptmx -> pts/ptmx
drwxr-xr-x. 2 root root      0 Aug  6 21:35 pts
crw-rw-rw-. 1 root root   1, 8 Aug  6 21:35 random
drwxrwxrwt. 2 root root     40 Aug  6 21:35 shm
lrwxrwxrwx. 1 root root     15 Aug  6 21:35 stderr -> /proc/self/fd/2
lrwxrwxrwx. 1 root root     15 Aug  6 21:35 stdin -> /proc/self/fd/0
lrwxrwxrwx. 1 root root     15 Aug  6 21:35 stdout -> /proc/self/fd/1
crw-rw-rw-. 1 root root   5, 0 Aug  6 21:35 tty
crw-rw-rw-. 1 root root   1, 9 Aug  6 21:35 urandom
crw-rw-rw-. 1 root root   1, 5 Aug  6 21:35 zero
sh-5.0#
```

The config.json snippet for this:
```
 "devices": [
      {
        "path": "/dev/mynull",
        "type": "c",
        "major": 1,
        "minor": 3,
        "fileMode": 8630,
        "uid": 0,
        "gid": 0
      }
    ],

```